geode-core/src/main/java/org/apache/geode/internal/cache/InternalRegion.java

```diff
@@ -459,4 +459,7 @@ VersionedObjectList basicRemoveAll(Collection<Object> keys,
    * @return true if synchronization should be attempted
    */
   boolean shouldSyncForCrashedMember(InternalDistributedMember id);
+
+  void clearRegionLocally(RegionEventImpl regionEvent, boolean cacheWrite,
```

Comment:
I wonder if we should change the API here? Will that impact our backward compatibility?

Do we have workaround not to change InternalRegion.java?

Reply:
I remove the code changes in InternalRegion, change the clearRegionLocally into public, then the code can compile.

Reply:
Not sure, what you meant here...InternalRegion is an interface and I added method in that to have access of it in the implemented classes.